types/sequelize/index.d.ts

```diff
@@ -31,13 +31,12 @@
 
 // Based on original work by: samuelneff <https://github.com/samuelneff/sequelize-auto-ts/blob/master/lib/sequelize.d.ts>
 
-/// <reference types="validator" />
-
-
 import * as _ from "lodash";
 import Promise = require("bluebird");
 import * as cls from "continuation-local-storage"
 
+type ValidatorJS = typeof import('validator')
```

Comment:
Prefer to keep the same import way as other [imports](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/40192/files#diff-0e3744815a33a1dfe3285ef4f51888e9R36)
```suggestion
import * as ValidatorJS from "validator";
```